client/test/components/ContentNavigator/convert.test.ts

```diff
@@ -0,0 +1,466 @@
+// Copyright © 2023, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { convertNotebookToFlow } from "../../../src/components/ContentNavigator/convert";
+import { deepEqual } from "assert";
+import { workspace } from "vscode";
+
+const contentSASNotebook1 = [
```

Comment:
nit: just providing an idea, will it look better to make separate files in `client/testFixture` directory.
Say `test.sasnb`, `test.ipynb`, `test.step`, `test.flw` etc. And read these files in this test.
The test fixture files can be opened in real world applications thus indicate it tests real world files and make this test file cleaner.

Reply:
done in 4b45ea3cd0c6ebd1a21ab73e963182ae250eb717